### ISSUE TITLE
training is not normal

### ISSUE THREAD

**Issue by lfxx** [D-X-Y/landmark-detection#39]
I trained the 68pts net on **your style_datasets**.But things  were not going as expected:
Firstly,i  ran **300W_Cluster.sh** to cluster the 300W-orginal datasets into three groups,here is my running log:
[cluster_seed_1986_22-Aug-at-07-51-49.txt](https://github.com/D-X-Y/landmark-detection/files/3539755/cluster_seed_1986_22-Aug-at-07-51-49.txt)
Since the  cluster-00-03.lst is very small,i chose cluster-01-03.lst/cluster-02-03.lst as cycle_a_lists and cycle_b_lists.
Then,i ran **300W_CyCLE_128.sh**,i found the running log is not normal,so i stopped it manually.Here is my running log:
[seed-8335-26-Aug-at-02-18-08.log](https://github.com/D-X-Y/landmark-detection/files/3539795/seed-8335-26-Aug-at-02-18-08.log)
in the running log above,D-A and D-B 's value are always around 0.15,can not reduce,any suggestions?
Thanks@D-X-Y

**Comment by D-X-Y** [comment]
Training GAN could sometimes be unstable. Would you mind to visualize some images from the generator to see whether it is good or not? If good, I think 0.15 of D-A should be fine.

**Comment by lfxx** [comment]
> Training GAN could sometimes be unstable. Would you mind to visualize some images from the generator to see whether it is good or not? If good, I think 0.15 of D-A should be fine.

I don't know how to see the images from generator,BTW,I noticed that when use  300W_Base_Cluster.sh and 300W_Cluster.sh to cluster datasets,their results would be totally different.For cluster-00-03.lst,former will have 0.16 percentage while latter only have 0.01 percentage!

**Comment by D-X-Y** [comment]
Would you mind to let know how do you visualize these generated images?
Yes, `300W_Base_Cluster.sh' and `300W_Cluster.sh' use different hyper-parameters. I suggest use `300W_Cluster.sh', which one do you use? This is my log for the cluster scrip: https://drive.google.com/open?id=1DtUg0LoZXNuDxeqRtcRxrA4tjrN9rpJa

**Comment by lfxx** [comment]
> Would you mind to let know how do you visualize these generated images?
> Yes, `300W_Base_Cluster.sh' and `300W_Cluster.sh' use different hyper-parameters. I suggest use `300W_Cluster.sh', which one do you use? This is my log for the cluster scrip: https://drive.google.com/open?id=1DtUg0LoZXNuDxeqRtcRxrA4tjrN9rpJa
> You can also check my log for the loss of D at https://drive.google.com/open?id=1SZVJHl8tM0G5MOFQmCrx5mB6vFj-aAFu , which decreased to 0.001 finally.

Yes.I visualize the images with the visual_freq=500 option on.I use 300W_Cluster.sh to cluster your style datasets,below is my result after 110 epoch:
![fake_A](https://user-images.githubusercontent.com/27956643/63735618-029c2700-c8b3-11e9-99a5-1173392c44a5.png)
![fake_B](https://user-images.githubusercontent.com/27956643/63735624-05971780-c8b3-11e9-8310-1864cd1f7210.png)
![idt_A](https://user-images.githubusercontent.com/27956643/63735659-1cd60500-c8b3-11e9-9241-f52d5b51b550.png)
![idt_B](https://user-images.githubusercontent.com/27956643/63735668-219ab900-c8b3-11e9-8c9b-b0057a4c6c01.png)
![real_A](https://user-images.githubusercontent.com/27956643/63735672-2495a980-c8b3-11e9-90b2-ad49402041eb.png)
![real_B](https://user-images.githubusercontent.com/27956643/63735678-26f80380-c8b3-11e9-9bcd-b633a79ae949.png)
![rec_A](https://user-images.githubusercontent.com/27956643/63735688-311a0200-c8b3-11e9-8d74-d4630e039ecd.png)
![rec_B](https://user-images.githubusercontent.com/27956643/63735691-337c5c00-c8b3-11e9-9baf-0e1bd1b0c14b.png)
[seed-4538-26-Aug-at-08-48-45.log](https://github.com/D-X-Y/landmark-detection/files/3543958/seed-4538-26-Aug-at-08-48-45.log)
seems like the net didn't learn anything!would somethings wrong on the train procedure?

**Comment by D-X-Y** [comment]
I'm not sure, the loss values in your log are quite different from my log.

**Comment by lfxx** [comment]
Ok,i will continue working on it,thanks!
